src/pages/task/metadata/Metadata.stories.tsx

```diff
@@ -1,98 +1,111 @@
+import { MockedProvider } from "@apollo/client/testing";
 import styled from "@emotion/styled";
+import { StoryObj } from "@storybook/react";
 import { MetStatus, RequiredStatus } from "gql/generated/types";
-import { Metadata } from "./index";
+import { Metadata, MetadataProps } from "./index";
 import { taskQuery } from "./taskData";
 
-export const Base = (args) => (
-  <Container>
-    <Metadata
-      {...args}
-      task={taskQuery.task}
-      taskId={taskQuery.task.id}
-      error={null}
-    />
-  </Container>
-);
+export default {
+  title: "Pages/Task/Metadata",
+  component: Metadata,
+  decorators: [
+    (Story: () => JSX.Element) => (
+      <MockedProvider>
+        <Story />
+      </MockedProvider>
+    ),
+  ],
+};
 
-export const WithDependencies = (args) => (
-  <Container>
-    <Metadata
-      {...args}
-      task={{
-        ...taskQuery.task,
-        dependsOn: [
-          {
-            buildVariant: "ubuntu1604",
-            metStatus: MetStatus.Unmet,
-            name: "Some dep",
-            requiredStatus: RequiredStatus.MustSucceed,
-            taskId: "some_task_id_1",
-          },
-          {
-            buildVariant: "ubuntu1604",
-            metStatus: MetStatus.Pending,
-            name: "Some dep",
-            requiredStatus: RequiredStatus.MustFinish,
-            taskId: "some_task_id_2",
-          },
-          {
-            buildVariant: "ubuntu1604",
-            metStatus: MetStatus.Met,
-            name: "Some dep",
-            requiredStatus: RequiredStatus.MustFail,
-            taskId: "some_task_id_3",
-          },
-          {
-            buildVariant: "ubuntu1604",
-            metStatus: MetStatus.Started,
-            name: "Some dep",
-            requiredStatus: RequiredStatus.MustFail,
-            taskId: "some_task_id_4",
-          },
-        ],
-      }}
-      taskId={taskQuery.task.id}
-      error={null}
-    />
-  </Container>
-);
+export const Default: StoryObj<MetadataProps> = {
```

Comment:
Can we use 
```suggestion
export const Default: StoryObj<typeof Metadata> = {
```
instead of exporting metadata props